samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -38,6 +38,15 @@ Configuration
 
 |config|
 
+This sample configuration is split into the following two files:
+
+* generic configuration is available in :file:`prj.conf` file
+* board specific configuration is avaialbe in :file:`boards/<BOARD>.conf` file
+
+Board specific configuration involves configuring the Bluetooth LE controller configuration.
```

Comment:
`Board specific configuration involves configuring the Bluetooth LE controller`

Reply:
Removed 'controller' word.

Reply:
Why remove the 'controller' word?
I think this sentence should be: Board-specific configuration involves configuring the Bluetooth LE controller.
Right?

Reply:
@mia-ko I've put wrong word in comment. The removed word is ` configuration`.